MAINT-48685: Disable unneeded csrf check for some read and front actions

hakermi opened the pull request:
Disable uneeded csrf check for some read and front actions to not end with csrf expire issue for no worthy reasons

ahamdi reviewed (changes requested):
Could you reset the formatting of those files, I can not find the exact changes in some files of this PR

hakermi committed:
MAINT-48685: Disable unneeded csrf check for some read and front actions

Disable uneeded csrf check for some read and front actions to not end with csrf expire issue for no worthy reasons

hakermi commented:
> Could you reset the formatting of those files, I can not find the exact changes in some files of this PR

I didn't' change the formatting of these files, it's interpreted in github that way
